Source/SwiftLintFramework/Rules/Lint/BalancedXCTestLifecycleRule.swift

```diff
@@ -150,13 +165,12 @@ private extension BalancedXCTestLifecycleRule {
 }
 
 private extension ClassDeclSyntax {
```

Comment:
Can be moved into `SwiftSyntax+SwiftLint.swift` to avoid the copy in `EmptyXCTestMethodRule`.